mmp/src/main/java/org/iit/healthcare/mmp/patientmodule/pages/ViewInformationPage.java

```diff
@@ -0,0 +1,68 @@
+package org.iit.healthcare.mmp.patientmodule.pages;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.iit.healthcare.mmp.helper.HelperClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class ViewInformationPage {
+
+	private WebDriver driver;
+	private HelperClass helperobj;
+	
+	public ViewInformationPage(WebDriver driver) {
+		this.driver = driver;
+	}
+	
+	public void ViewInformationContent() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.